Bottom: 36df384d99799c6351714fbc2e5e83677d302823
Top:    44bf0d63c5a4ab0cb1483e03c3e262d256dcd6d8
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-04-14 15:04:58 +0800

Refresh of debug

---

diff --git a/drivers/virtio/virtio_pci_modern.c b/drivers/virtio/virtio_pci_modern.c
index 39e42bd..2de3207 100644
--- a/drivers/virtio/virtio_pci_modern.c
+++ b/drivers/virtio/virtio_pci_modern.c
@@ -233,6 +233,8 @@ static void vp_set_coalesce(struct virtio_device *vdev, int n,
 {
 	struct virtio_pci_device *vp_dev = to_vp_device(vdev);
 
+	printk("index %d c %d u %d\n",
+		n, coalesce_count, coalesce_us);
 	iowrite16(n, &vp_dev->common->queue_select);
 	iowrite32(coalesce_count, &vp_dev->common->queue_coalesce_count);
 	iowrite32(coalesce_us, &vp_dev->common->queue_coalesce_us);
